packages/tooltip/src/Tooltip.tsx

```diff
@@ -0,0 +1,301 @@
+import React, { useState, useRef } from 'react';
+import PropTypes from 'prop-types';
+import Popover, { PopoverProps, Align, Justify } from '@leafygreen-ui/popover';
+import { useEventListener } from '@leafygreen-ui/hooks';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { uiColors } from '@leafygreen-ui/palette';
+import { transparentize } from 'polished';
+import debounce from 'lodash/debounce';
+import { trianglePosition } from './tooltipUtils';
+
+export const TriggerEvent = {
+  Hover: 'hover',
+  Click: 'click',
+} as const;
+
+type TriggerEvent = typeof TriggerEvent[keyof typeof TriggerEvent];
+
+export const Variant = {
+  Light: 'light',
+  Dark: 'dark',
+} as const;
+
+type Variant = typeof Variant[keyof typeof Variant];
+
+const baseStyles = css`
+  font-size: 14px;
+  line-height: 20px;
+  padding: 14px 16px 15px 16px;
```

Comment:
This... seems wrong. Probably should be `14px 16px` or just `16px`.

Reply:
Based on the invision spec! Lmk which direction to take and will apply changes

Reply:
Occasionally for various reasons, those can be slightly inaccurate when indicating the exact value (e.g. when centering an object, sometimes it will snap to the nearest pixel value rather than use sub-pixels). Let's run with `14px 16px` as it looks like that's the intent.

Reply:
👍